.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+# Cut a release whenever a new tag is pushed to the repo.
+# You should use an annotated tag, like `git tag -a v1.2.3`.
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Prepare workspace snippet
+        run: .github/workflows/workspace_snippet.sh ${{ env.GITHUB_REF_NAME }} > release_notes.txt
```

Comment:
Thanks @jvandew, this is really cool. Just a small question: does `${{ env.GITHUB_REF_NAME }}` needs to be passed here? Cannot find where it's taken in the script, looks like it's read from env.

Reply:
aha you appear to be correct, nice catch!